tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/InteractiveFlowTests.cs

```diff
@@ -366,5 +375,183 @@ private SeleniumWebUI CreateSeleniumCustomWebUI(LabUser user, Prompt prompt, boo
                 driver.PerformLogin(user, prompt, withLoginHint, adfsOnly);
             }, TestContext);
         }
+
+        #region Azure AD Kerberos Feature Tests
+        [TestMethod]
+        public async Task Kerberos_Interactive_AADAsync()
+        {
+            // Arrange
+            LabResponse labResponse = await LabUserHelper.GetDefaultUserAsync().ConfigureAwait(false);
+            await KerberosRunTestForUserAsync(labResponse, KerberosTicketContainer.IdToken).ConfigureAwait(false);
+            await KerberosRunTestForUserAsync(labResponse, KerberosTicketContainer.AccessToken).ConfigureAwait(false);
+        }
+
+#if DESKTOP // no point in running these tests on NetCore - the code path is similar
+        [TestMethod]
+        public async Task Kerberos_Interactive_AdfsV4_FederatedAsync()
```

Comment:
I see that you are already running ROPC tests with ADFSV4 user. Maybe this is enough? Interactive tests use UI automation and are quite lengthy and somewhat unreliable. I'd think `Kerberos_Interactive_AADAsync` provides enough coverage?